### ISSUE TITLE
Missing interpretations for `tschuprows_t`, `riskratio`

### ISSUE THREAD

**Issue by rempsyc** [easystats/effectsize#543]
Initially mentioned in easystats/report#311. Contrary to other effect size types, `tschuprows_t` and `riskratio` don't have accompanying interpretations. Additionally, for `cohens_w`, there is a `max_possible not found` error, although I'm not sure if that's because I'm feeding it the wrong type of object.

``` r
library(effectsize)
packageVersion("effectsize")
#> [1] '0.8.2.4'

M <- as.table(rbind(c(762, 327, 468), c(484, 239, 477)))
dimnames(M) <- list(gender = c("F", "M"), party = c("Democrat","Independent", "Republican"))
(x <- chisq.test(M)) 
#> 
#>  Pearson's Chi-squared test
#> 
#> data:  M
#> X-squared = 30.07, df = 2, p-value = 2.954e-07

# tschuprows_t has no interpretation in effectsize
y <- effectsize::effectsize(x, type = "tschuprows_t")
y
#> Tschuprow's T |       95% CI
#> ----------------------------
#> 0.09          | [0.06, 1.00]
#> 
#> - One-sided CIs: upper bound fixed at [1.00].
y <- effectsize::interpret(y, rules = "funder2019")
y$Interpretation
#> NULL
names(y)
#> [1] "Tschuprows_t" "CI"           "CI_low"       "CI_high"
y
#> Tschuprow's T |       95% CI
#> ----------------------------
#> 0.09          | [0.06, 1.00]
#> 
#> - One-sided CIs: upper bound fixed at [1.00].

# max_possible not found for cohens_w
effectsize::effectsize(x, type = "cohens_w")
#> Error in f(chisq = .chisq(Obs, Exp), n = sum(Obs), nrow = nr, ncol = nc, : object 'max_possible' not found

# Change dataset for "Error: Phi is not appropriate for non-2x2 tables."
(x <- chisq.test(RCT_table)) 
#> Error in is.data.frame(x): object 'RCT_table' not found

# riskratio has no interpretation in effectsize
y <- effectsize::effectsize(x, type = "riskratio")
#> Error: Risk ratio only available for 2-by-2 contingency tables
y
#> Tschuprow's T |       95% CI
#> ----------------------------
#> 0.09          | [0.06, 1.00]
#> 
#> - One-sided CIs: upper bound fixed at [1.00].
y <- effectsize::interpret(y, rules = "chen2010")
y$Interpretation
#> NULL
names(y)
#> [1] "Tschuprows_t" "CI"           "CI_low"       "CI_high"
y
#> Tschuprow's T |       95% CI
#> ----------------------------
#> 0.09          | [0.06, 1.00]
#> 
#> - One-sided CIs: upper bound fixed at [1.00].
```

<sup>Created on 2022-12-20 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by mattansb** [comment]
Is `report` scheduled to be updated on CRAN soon?

**Comment by rempsyc** [comment]
It's been 17.1 weeks according to the [dashboard](https://easystats.github.io/dashboard/dashboard.html), but I suppose it's not really a priority. Just to put it on the map.

**Comment by DominiqueMakowski** [comment]
I can push it to cran if need be

**Comment by IndrajeetPatil** [comment]
> This year's winter break will be from Dec 22, 2022 to Jan 5, 2023.

**Comment by IndrajeetPatil** [comment]
So CRAN update for _report_ can only be updated sometime after 6th of Jan, 2023.

**Comment by mattansb** [comment]
Fixed:

``` r
library(effectsize)

M <- as.table(rbind(c(762, 327, 468), c(484, 239, 477)))
Xsq <- chisq.test(M)

Tt <- effectsize(Xsq, type = "tschuprows_t")
W <- effectsize(Xsq, type = "cohens_w")


effectsize::interpret(Tt, rules = "funder2019")
#> Tschuprow's T |       95% CI | Interpretation
#> ---------------------------------------------
#> 0.09          | [0.06, 1.00] |     very small
#> 
#> - One-sided CIs: upper bound fixed at [1.00].
#> - Interpretation rule: funder2019
effectsize::interpret(W, rules = "funder2019")
#> Cohen's w |       95% CI | Interpretation
#> -----------------------------------------
#> 0.10      | [0.07, 1.00] |          small
#> 
#> - One-sided CIs: upper bound fixed at [1.00].
#> - Interpretation rule: funder2019
```

<sup>Created on 2022-12-21 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>


Note that risk ratios aren't supported - I am not aware of any rules of thumb for interpreting RR.
